File: AkitaModel/Portal/TestSummary.cs
namespace Skyware.Lis.AkitaModel.Portal;

/// <summary>
/// Entry of the patient's file.
/// </summary>
public class TestSummary
{

    /// <summary>
    /// The Test this summary is for.
    /// </summary>
    public Test Test { get; set; }

    /// <summary>
    /// Total number of test results over time.
    /// </summary>
    public int NumberOfResults { get; set; }

    /// <summary>
    /// Latest test result.
    /// </summary>
    public TestResult LastResult { get; set; }

}

File: AkitaMotelTS/src/Portal/TestSummary.ts
/**
 * Skyware.Lis.AkitaModel.Portal
 */

import { Test } from '../Test';
import { TestResult } from './TestResult';

/**
 * Entry of the patient's file.
 */
export class TestSummary {
  /**
   * The Test this summary is for.
   */
  test?: Test;

  /**
   * Total number of test results over time.
   */
  numberOfResults!: number;

  /**
   * Latest test result.
   */
  lastResult?: TestResult;
}
